apps/web/src/pages/Users/AdminUserProfilePage/AdminUserProfilePage.tsx

```diff
@@ -29,7 +29,8 @@ export const AdminUserProfile = ({ user }: AdminUserProfileProps) => {
         data-h2-container="base(center, large, x1) p-tablet(center, large, x2)"
         data-h2-text-align="base(right)"
       >
-        <UserProfilePrintButton user={user}>
+        {/* leaving anonymous for testing */}
+        <UserProfilePrintButton user={user} anonymous>
```

Comment:
are you going to remove this for merging in?

Reply:
Yes I'll remove this before merging!